src/release.js

```diff
@@ -32,8 +32,11 @@
  *************************************************************************/
 
 /**
- * WV RELEASE INFORMATION
- */
+* WV RELEASE INFORMATION
+*/
+// Fixes error building the I18n
+var WV_NAME = "WME Validator";
+var WV_SHORTNAME = "Validator";
```

Comment:
There is a commit https://github.com/WMEValidator/i18n/commit/1545a70a16cd577b1a18348417b888b188ed3777 removing those constants.

Reply:
I didn't see that, oops.
So need to remove this again....

Reply:
No problem, I merged without this commit ;)

Reply:
Great! Thanks 😄